app/views/users/_edit_profile.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_tag path, method: :patch do %>
+  <%= label_tag :name %>
+  <%= text_field_tag :name, name_value %>
+
+  <%= label_tag :street_address %>
+  <%= text_field_tag :street_address, address_value %>
+
+  <%= label_tag :city %>
+  <%= text_field_tag :city, city_value %>
+
+  <%= label_tag :state %>
+  <%= text_field_tag :state, state_value %>
+
+  <%= label_tag :zip %>
+  <%= text_field_tag :zip, zip_value %>
+
+  <%= label_tag :email %>
+  <%= text_field_tag :email, email_value %>
+
+  <%= hidden_field_tag :old_email, original_email_value %>
+
+  <%= yield %>
```

Comment:
Would love to learn more about what this yield is doing

Reply:
@jdcarey128 Did you get the yield working in the partial?